edge-hub/docker/windows/amd64/Dockerfile

```diff
@@ -12,9 +12,4 @@ EXPOSE 8883/tcp
 EXPOSE 5671/tcp
 EXPOSE 443/tcp
 
-USER ContainerAdministrator
```

Comment:
Why don't we need ContainerUser declared here?

Reply:
Sorry I meant removal of USER edgehubuser

Reply:
Not sure what the question is here.. we don't need to add the edgeHubuser or run as the EdgeHubUser. We simply run as ContainerUser.